iceoryx-sys/build.rs

```diff
@@ -110,6 +132,7 @@ fn main() -> std::io::Result<()> {
     extract_archive(&iceoryx_archive_dir, &iceoryx_source_dir, ICEORYX_VERSION)?;
 
     make_and_install(
+        &iceoryx_archive_dir,
```

Comment:
With this change it makes semantically more sense to rename this to `iceoryx_cpp_dir`